ui/app/public/index.html

```diff
@@ -9,7 +9,7 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-MSGFSQQEJ3"></script>
     <script>
-      if (window.location.href === "dex.sifchain.finance") {
+      if (window.location.host === "dex.sifchain.finance") {
```

Comment:
shouldn't this be hostname, not host?  (I thought host included :port - https://developer.mozilla.org/en-US/docs/Web/API/Location)

Reply:
`window.location.host` works. Go to dex.sifchain.finance console, then input `console.log(window.location.host)`. Will return desired `dex.sifchain.finance`